GenderPayGap.UnitTests/GenderPayGap.WebUI.Tests/TestHelpers/NewUiTestHelper.cs

```diff
@@ -65,7 +65,7 @@ public static T GetController<T>(long userId = 0,  Dictionary<string, StringValu
             DIContainer = BuildContainerIoC(dbObjects);
 
             //Create Inversion of Control container
-            MvcApplication.ContainerIoC = DIContainer;
+            Global.ContainerIoC = DIContainer;
```

Comment:
When you get a chance, could we have a brief chat to discuss this change please? I'd like to know what's going on, the reason behind the change and the implications of moving this IocContainer